config/management/README.md

```diff
@@ -1,190 +1,14 @@
-
-### The Association
-
-* The association will publish a layout containing the distinct names and roles
-  of the participants, this is placed into a common namespace:
-```
-cargo run -p libra-management -- \
-    set-layout \
-    --path PATH_TO_LAYOUT \
-    --backend 'backend=github;repository_owner=REPOSITORY_OWNER;repository=REPOSITORY;token=PATH_TO_GITHUB_TOKEN;namespace=common'
-```
-* Each Member of the Association will upload their key to GitHub:
-```
-cargo run -p libra-management -- \
-    libra-root-key \
-    --local 'backend=vault;server=URL;token=PATH_TO_VAULT_TOKEN' \
-    --remote 'backend=github;repository_owner=REPOSITORY_OWNER;repository=REPOSITORY;token=PATH_TO_GITHUB_TOKEN;namespace=NAME'
+### The Tools
 ```
-
-The layout is a toml configuration file of the following format:
-```
-[operator] = ["alice", "bob"]
-[owner] = ["carol", "dave"]
-[libra_root] = ["erin"]
-```
-where each field maps to a role as described in this document.
-
-### Validator Owners
-
-* Each Validator Owner member will upload their key to GitHub:
-```
-cargo run -p libra-management -- \
-    owner-key \
-    --local 'backend=vault;server=URL;token=PATH_TO_VAULT_TOKEN' \
-    --remote 'backend=github;repository_owner=REPOSITORY_OWNER;repository=REPOSITORY;token=PATH_TO_GITHUB_TOKEN;namespace=NAME'
-```
-
-* Each validator owner will select the validator operator responsible for
-  operating the validator node. This selection is done by specifying the name of
-  the validator operator (as registered in the shared Github):
-```
-cargo run -p libra-management -- \
-    set-operator \
-    --operator-name OPERATOR_NAME \
-    --local 'backend=vault;server=URL;token=PATH_TO_VAULT_TOKEN' \
-    --remote 'backend=github;repository_owner=REPOSITORY_OWNER;repository=REPOSITORY;token=PATH_TO_GITHUB_TOKEN;namespace=NAME'
-```
-
-### Validator Operators
-
-* Each Validator Operator member will upload their key to GitHub:
-```
-cargo run -p libra-management -- \
-    operator-key \
-    --local 'backend=vault;server=URL;token=PATH_TO_VAULT_TOKEN' \
-    --remote 'backend=github;repository_owner=REPOSITORY_OWNER;repository=REPOSITORY;token=PATH_TO_GITHUB_TOKEN;namespace=NAME'
-```
-* For each validator managed by an operator, the operator will upload a signed
-  validator-config. The owner corresponds to the name of the validator owner (as
-  registered in the shared Github). The namespace in GitHub correlates to the
-  operator namespace:
-```
-cargo run -p libra-management -- \
-    validator-config \
-    --owner-name OWNER_NAME \
-    --validator-address '/dns/DNS/tcp/PORT' \
-    --fullnode-address '/dns/DNS/tcp/PORT' \
-    --local 'backend=vault;server=URL;token=PATH_TO_VAULT_TOKEN' \
-    --remote 'backend=github;repository_owner=REPOSITORY_OWNER;repository=REPOSITORY;token=PATH_TO_GITHUB_TOKEN;namespace=NAME'
+libra-management
+|-> libra-genesis-tool = A tool for managing genesis of a Libra blockchain.
```

Comment:
nit: maybe "A tool for performing the genesis ceremony for the Libra blockchain"